src/Config.ts

```diff
@@ -1,14 +1,16 @@
 import path from 'path';
 import fs from 'fs';
-import DiskCache from "./DiskCache.js";
-import Workdays from "./Workdays.js";
+import DiskCache from './DiskCache';
+import Workdays from './Workdays';
 
-import generate from './lib/generate.js';
-import ConfigInterface from './interfaces/ConfigInterface.js';
-import IOBase from './IOBase.js';
+import generate from './lib/generate';
+import ConfigInterface from './interfaces/ConfigInterface';
+import IOBase from './IOBase';
+import ConfigNotFoundError from './errors/ConfigNotFoundError';
 
 class Config extends IOBase {
   private workdays: Workdays;
+
   private cache: DiskCache;
 
   constructor(cache: DiskCache, workdays: Workdays, baseDir = './.config') {
```

Comment:
Standaard uitgaan van productie omgeving
```suggestion
  constructor(cache: DiskCache, workdays: Workdays, baseDir = '/config') {
```